model/description-model.js

```diff
@@ -0,0 +1,14 @@
+const fs =  require("node:fs/promises");
+
+const fetchDescription = () => {
+    return fs.readFile('./endpoints.json', 'utf-8', (err, data) => {
+        if(err){
+            console.log(err)
+            return;
+        }else{
+            return data
+        }
+    })
+};
+
+module.exports = { fetchDescription };
```

Comment:
You don't need a separate model for this as we're not dealing with the database. File can be required in the controller and don't necessarily need fs either.